src/servers/main.cc

```diff
@@ -802,8 +802,8 @@ Parse(TRTSERVER_ServerOptions* server_options, int argc, char** argv)
 #ifdef TRTIS_ENABLE_HTTP
   http_port_ = http_port;
   http_health_port_ = http_health_port;
-  http_ports_ = {http_port_, http_health_port_, http_port_, http_port_,
-                 http_port_};
+  http_ports_ = {http_port_, http_health_port_, http_port_,
+                 http_port_, http_port_,        http_port_};
```

Comment:
format

Reply:
This was caused by the python format command. Should I remove the extra spacing even though our formatter would revert back to this every time it is run?

Reply:
No, if the formatter does it then leave it... I didn't know it did that...